frontend/app/components/side-nav-bar.js

```diff
@@ -37,7 +37,7 @@ export default class SideNavBar extends Component {
     else {
       this.router
         .transitionTo("teams.index", {
-          queryParams: { team_id: team.id, folder_id: null }
+          queryParams: { team_id: team.id, folder_id: undefined, q: undefined }
```

Comment:
does `null` not work here?

Reply:
It does work. But I replaced it for consistency.